src/lib/middleware/hardcoded_redirects.coffee

```diff
@@ -44,6 +44,7 @@ redirects =
   '/following/profiles': '/user/saves#galleries-institutions'
   '/artsy.net/artwork/marilyn-minter-miley': '/artwork/marilyn-minter-miley'
   '/article/artsy-editorial-the-year-in-art-2016': '/2016-year-in-art'
+  '/gallery-insights': 'https://partners.artsy.net/'
```

Comment:
This _might_ work, but currently `/gallery-insights` responds with some sort of actual page, which probably exists somewhere in Force. So while this line might work here, we should investigate what gallery insights currently is, if there's cleanup we can do there (code removal!), etc.

Reply:
As noted in the comments, we are going to remove the app code separately -- it uses modifications of the `articles` and `article` app, so removing this app will involve picking pieces out of a number of different components and apps.